src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -172,6 +173,11 @@ const PaymentAgentWithdraw = (() => {
 
     const onLoad = () => {
         BinarySocket.wait('get_account_status').then((data) => {
+            if (!Client.get('currency')) {
```

Comment:
why do we need to wait for account status to check currency? don't we already get that from authorize? and this page only loads when authorize response is received